VMBackup/main/handle.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
please revert this to bash shell. For freeBSD we are suggesting customers to install bash. We have plans to move this to bourne shell in few days. It has to be tested with all the supported distros.

Reply:
Unfortunately, NSBSD is not FreeBSD, Bash is not available and I can't add it. If you already plan to move to sh, I can wait and merge. At first look there is no bash specific syntax in the script.

Reply:
got it. I will merge the changes with bourne shell, as we plan to support bourne shell in next release